pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,32 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return DatetimeIndex as object ndarray of datetime.datetime objects.
```

Comment:
Return DatetimeIndex as an object ndarray

Reply:
Thanks. Fixed it.